dev/statement_serializer.h

```diff
@@ -37,11 +38,71 @@
 #include "triggers.h"
 #include "table_type.h"
 #include "index.h"
+#include "util.h"
 
 namespace sqlite_orm {
 
     namespace internal {
 
+        inline std::string
+        serialize_identifier(const std::string& qualifier, const std::string& identifier, const std::string& alias) {
+            std::string quoted = quote_identifier(identifier);
+            if(!qualifier.empty()) {
+                quoted = quote_identifier(qualifier) + "." + move(quoted);
+            }
+            if(!alias.empty()) {
+                quoted += " " + quote_identifier(alias);
+            }
+            return quoted;
+        }
+
+        inline std::string serialize_identifier(const std::string& identifier, const std::string& alias) {
+            std::string quoted = quote_identifier(identifier);
```

Comment:
Yes, true, however it's a kind of a shortcut - why calling the other function with useless temporary objects?

Reply:
because if you decide to improve you serialize algorithm inside these functions you'll need to improve it in several times. You'll do copy and paste code. But if you write one thing at one place then you'll never face copy and paste code